app/src/main/java/com/jxc/app/models/Address.java

```diff
@@ -0,0 +1,15 @@
+package com.jxc.app.models;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
```

Comment:
Dans ma boite on préfère faire des classes immuables, qu'il est possible de créer avec `@Value`. 

Voir ici pourquoi : 
- http://javaexplorer03.blogspot.com/2016/12/advantages-and-disadvantages-immutable.html
- https://hackernoon.com/5-benefits-of-immutable-objects-worth-considering-for-your-next-project-f98e7e85b6ac